metricbeat/module/kubernetes/_meta/test/docs/01_playground/playground-ubuntu.yaml

```diff
@@ -0,0 +1,79 @@
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: metricbeat-kube
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: metricbeat
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - nodes/stats
+  - nodes/metrics
+  - nodes/log
+  - nodes/spec
+  - nodes/proxy
+  verbs:
+  - get
+- apiGroups: [""] 
+  resources: ["pods", "nodes"]
+  verbs: ["get", "watch", "list"]
+- apiGroups: ["extensions", "apps"]
+  resources: ["deployments", "replicasets", "statefulsets"]
+  verbs: ["get", "list", "watch"]
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: metricbeat-kube
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: metricbeat
+subjects:
+- kind: ServiceAccount
+  name: metricbeat-kube
+  namespace: default
```

Comment:
It could be good to generate these files as we generate `deploy/kubernetes/metricbeat-kubernetes.yaml`, and reusing specific resources when possible, this way we could also check for example if permissions are well configured.

Reply:
agree, although generation gives me the goosebumps when dealing with multiple "projects", but as long as that doc generation lives inside metricbeat it sounds good.

Anyway I've been thinking for some time on using aggregation rules for RBAC instead of a huge generated yaml element. I'll create an issue for that.